components/trades/create-trade.tsx

```
@@ -0,0 +1,24 @@
+import TradeComposer from '@/components/trades/TradeComposer';
+
+import { useSession } from '@/hooks/auth-hooks';
+
+export default function CreateTrade() {
+    const { data: session } = useSession();
+
+    return (
+        <>
+            <div className="flex min-h-screen flex-col items-center justify-center gap-1 p-4 md:p-0">
+                <div className="flex w-full max-w-[750px] justify-start">
+                    <button className="flex items-center gap-1 rounded-lg bg-[#F0F0F0] px-2 py-1.5 dark:bg-[#1A1A1A]">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)